Issue in Postgress database with  blob column

Hi, when I create a Entity with blob attribute  I got an error on create entity

    SQL Error: 0, SQLState: 42804
    [ERROR] org.hibernate.engine.jdbc.spi.SqlExceptionHelper - ERROR: the column ┬½archivo┬╗ 
    is of type bytea but the expression is  oid type....

jhipster with Mysql runs fine   

I replace @Lob with deprecated type
    @Type(type="org.hibernate.type.BinaryType")

and work well but  in this reference;
[link to stackoverflow.com!]( http://stackoverflow.com/questions/3677380/proper-hibernate-annotation-for-byte)  I read a better aproach to do this but when I put the sentence 
**hibernate.jdbc.use_streams_for_binary=false** in  application-dev.yml
hibernate properties still won´t work
   

Thanks in advance

--- a/app/index.js
+++ b/app/index.js
@@ -692,6 +692,9 @@ JhipsterGenerator.prototype.app = function app() {
     this.template('src/main/java/package/domain/util/_ISO8601LocalDateDeserializer.java', javaDir + 'domain/util/ISO8601LocalDateDeserializer.java', this, {});
     if (this.databaseType == "sql") {
         this.template('src/main/java/package/domain/util/_FixedH2Dialect.java', javaDir + 'domain/util/FixedH2Dialect.java', this, {});
+        if (this.prodDatabaseType == 'postgresql') {
+            this.template('src/main/java/package/domain/util/_FixedPostgreSQL82Dialect.java', javaDir + 'domain/util/FixedPostgreSQL82Dialect.java', this, {});
+        }
     }
 
     if (this.searchEngine == 'elasticsearch') {

--- a/app/templates/src/main/java/package/domain/util/_FixedPostgreSQL82Dialect.java
+++ b/app/templates/src/main/java/package/domain/util/_FixedPostgreSQL82Dialect.java
@@ -1,0 +1,23 @@
+package <%=packageName%>.domain.util;
+
+import org.hibernate.dialect.PostgreSQL82Dialect;
+import org.hibernate.type.descriptor.sql.BinaryTypeDescriptor;
+import org.hibernate.type.descriptor.sql.SqlTypeDescriptor;
+
+import java.sql.Types;
+
+public class FixedPostgreSQL82Dialect extends PostgreSQL82Dialect {
+
+    public FixedPostgreSQL82Dialect() {
+        super();
+        registerColumnType(Types.BLOB, "bytea");
+    }
+
+    @Override
+    public SqlTypeDescriptor remapSqlTypeDescriptor(SqlTypeDescriptor sqlTypeDescriptor) {
+        if (sqlTypeDescriptor.getSqlType() == java.sql.Types.BLOB) {
+            return BinaryTypeDescriptor.INSTANCE;
+        }
+        return super.remapSqlTypeDescriptor(sqlTypeDescriptor);
+    }
+}

--- a/app/templates/src/main/resources/config/_application-dev.yml
+++ b/app/templates/src/main/resources/config/_application-dev.yml
@@ -45,7 +45,7 @@ spring:
         <%_ if (devDatabaseType == 'mysql') { _%>
         database-platform: org.hibernate.dialect.MySQLInnoDBDialect
         <%_ } else if (devDatabaseType == 'postgresql') { _%>
-        database-platform: org.hibernate.dialect.PostgreSQL82Dialect
+        database-platform: <%=packageName%>.domain.util.FixedPostgreSQL82Dialect
         <%_ } else if (devDatabaseType == 'oracle') { _%>
         database-platform: org.hibernate.dialect.Oracle10gDialect
         <%_ } else { _%>

--- a/app/templates/src/main/resources/config/_application-prod.yml
+++ b/app/templates/src/main/resources/config/_application-prod.yml
@@ -47,7 +47,7 @@ spring:
         <%_ if (prodDatabaseType == 'mysql') { _%>
         database-platform: org.hibernate.dialect.MySQLInnoDBDialect
         <%_ } else if (prodDatabaseType == 'postgresql') { _%>
-        database-platform: org.hibernate.dialect.PostgreSQL82Dialect
+        database-platform: <%=packageName%>.domain.util.FixedPostgreSQL82Dialect
         <%_ } else if (prodDatabaseType == 'oracle') { _%>
         database-platform: org.hibernate.dialect.Oracle10gDialect
         <%_ } else { _%>

